### ISSUE TITLE
[2.0] On mobile, the Language and Accessibility dropdowns have misaligned font

### ISSUE THREAD

**Issue by doradocodes** [processing/p5.js-website#1480]
### Most appropriate sections of the p5.js website?

Home

### What is your operating system?

Mac OS

### Web browser and version

Chrome

### Actual Behavior

1. Open the [beta.p5js.org](https://beta.p5js.org/) website on mobile view
2. See the "Language" and "Accessibility" dropdowns:
<img width="392" height="847" alt="Image" src="https://github.com/user-attachments/assets/2f3416c3-e48d-4a40-8eab-8d051d57f14f" />

### Expected Behavior

The "Language" and "Accessibility" dropdowns should have the equal font size as the Search bar. For the "Accessibility" dropdown, the text should not overlap with the background. 

<img width="467" height="1066" alt="Image" src="https://github.com/user-attachments/assets/3921bb0b-72a2-467b-8d28-6175400ed962" />

### Would you like to work on the issue?

No

**Comment by ksen0** [comment]
This affects the version 2 site specifically. If [this issue](https://github.com/processing/p5.js-website/issues/1426) is marked done, fix should target `main` branch. If that issue is NOT marked done, then target `2.0`. If in doubt, test the site locally and confirm if it.s the version2 or not.

**Comment by slash-init** [comment]
Hi! I was able to reproduce this bug locally and I think I've found the root cause. Could I be assigned to work on this issue?

**Comment by doradocodes** [comment]
Hi @slash-init, yes I will assign this issue to you. Thanks!

**Comment by slash-init** [comment]
Thanks @doradocodes ! I've opened a PR which resolves this issue.
